mmdet/models/detectors/base.py

```diff
@@ -391,3 +399,7 @@ def show_result(self,
             warnings.warn('show==False and out_file is not specified, only '
                           'result image will be returned')
             return img
+
+    def export(self, img, img_metas, export_name, **kwargs):
```

Comment:
```suggestion
    def export(self, img, img_metas, export_name, network = None, **kwargs):
        if network is not None:
            # the network parameter may be set using functools.partial
            self = network
```

Reply:
P.S. two more comments:
* The parameter may be `nncf_network` instead of `network`.  It will make more clear what happens.
* Absolutely another way to resolve the issue is to make the method `export` be intentionally static. In this case you will have to write only two explanations: in the base implementation of the method and in the `tools/export.py`. But IMHO this way is less pretty.